docs/proto/events.md

```diff
@@ -117,44 +117,44 @@ Represents a security violation that is emitted by the agent
 
 | Field | Type | Label | Description |
 | ----- | ---- | ----- | ----------- |
-| PolicyName | [string](#string) |  |  |
-| SupportID | [string](#string) |  |  |
-| Outcome | [string](#string) |  |  |
-| OutcomeReason | [string](#string) |  |  |
-| BlockingExceptionReason | [string](#string) |  |  |
-| Method | [string](#string) |  |  |
-| Protocol | [string](#string) |  |  |
-| XForwardedForHeaderValue | [string](#string) |  |  |
-| URI | [string](#string) |  |  |
-| Request | [string](#string) |  |  |
-| IsTruncated | [string](#string) |  |  |
-| RequestStatus | [string](#string) |  |  |
-| ResponseCode | [string](#string) |  |  |
-| ServerAddr | [string](#string) |  |  |
-| VSName | [string](#string) |  |  |
-| RemoteAddr | [string](#string) |  |  |
-| RemotePort | [string](#string) |  |  |
-| ServerPort | [string](#string) |  |  |
-| Violations | [string](#string) |  |  |
-| SubViolations | [string](#string) |  |  |
-| ViolationRating | [string](#string) |  |  |
-| SigSetNames | [string](#string) |  |  |
-| SigCVEs | [string](#string) |  |  |
-| ClientClass | [string](#string) |  |  |
-| ClientApplication | [string](#string) |  |  |
-| ClientApplicationVersion | [string](#string) |  |  |
-| Severity | [string](#string) |  |  |
-| ThreatCampaignNames | [string](#string) |  |  |
-| BotAnomalies | [string](#string) |  |  |
-| BotCategory | [string](#string) |  |  |
-| EnforcedBotAnomalies | [string](#string) |  |  |
-| BotSignatureName | [string](#string) |  |  |
-| ViolationsData | [ViolationData](#f5-nginx-agent-sdk-events-ViolationData) | repeated |  |
-| SystemID | [string](#string) |  |  |
-| InstanceTags | [string](#string) |  |  |
-| InstanceGroup | [string](#string) |  |  |
-| DisplayName | [string](#string) |  |  |
-| ParentHostname | [string](#string) |  |  |
+| PolicyName | [string](#string) |  | The name of the policy that triggered the security violation |
+| SupportID | [string](#string) |  | The unique support ID of the violation to be tracked with |
```

Comment:
Same here, should we be explicit about where the support ID originates from?
```suggestion
| SupportID | [string](#string) |  | The unique NGINX App Protect support ID of the violation, used for tracking purposes |
```

Reply:
Yep I think it's better to be explicit. agreed and applied 👍